frontend/javascripts/oxalis/view/action-bar/create_animation_modal.tsx

```diff
@@ -230,7 +230,8 @@ function CreateAnimationModal(props: Props) {
       cameraPosition: selectedCameraPosition,
     };
 
-    if (!validateAnimationOptions(colorLayer, boundingBox, meshes)) return;
+    const selectedColorLayer = colorLayers.find((layer) => layer.name === selectedColorLayerName)!;
```

Comment:
line 108 already defines `selectedColorLayer`, which should be usable here.

cf: line 108: `const selectedColorLayer = getLayerByName(dataset, selectedColorLayerName);`

Reply:
Thus, I'd suggest to remove this line

Reply:
good catch. done.